### PR TITLE
fix: fix package build

### LINKED ISSUE #2: ModulenotFoundError
I used:
- ubuntu 20.04. 
- Pycharm pro ( trial)
- create venv and install as your document:
` pip install casbin_mongoengine_adapter`

When I start app I received this message. Can you guys give me some suggestion:

```
Traceback (most recent call last):
  File "app.py", line 2, in <module>
    import casbin_mongoengine_adapter
ModuleNotFoundError: No module named 'casbin_mongoengine_adapter'
```

Update this is output when I check does package installed ?
```
$ python app.py 
Traceback (most recent call last):
  File "app.py", line 2, in <module>
    import casbin_mongoengine_adapter
ModuleNotFoundError: No module named 'casbin_mongoengine_adapter'

$ pip install casbin_mongoengine_adapter
Requirement already satisfied: casbin_mongoengine_adapter in ./venv/lib/python3.7/site-packages (0.1.0)
Requirement already satisfied: casbin in ./venv/lib/python3.7/site-packages (from casbin_mongoengine_adapter) (0.16.0)
Requirement already satisfied: mongoengine in ./venv/lib/python3.7/site-packages (from casbin_mongoengine_adapter) (0.22.1)
Requirement already satisfied: simpleeval>=0.9.10 in ./venv/lib/python3.7/site-packages (from casbin->casbin_mongoengine_adapter) (0.9.10)
Requirement already satisfied: pymongo<4.0,>=3.4 in ./venv/lib/python3.7/site-packages (from mongoengine->casbin_mongoengine_adapter) (3.11.3)
```

### PR DESCRIPTION
**Pull request by Zxilly**
FIX: #2

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def run(self):
     author_email=EMAIL,
     python_requires=REQUIRES_PYTHON,
     url=URL,
-    packages=find_packages(exclude=["tests", "casbin_mongoengine_adapter"]),
+    packages=find_packages(),
```
Comment (fenixpapu) [inline setup.py:106]: I'm wondering, should we use:
```
exclude=["tests"]
```
rather than remove all?

Comment (hsluoyz) [reply setup.py:106]: @Zxilly

Comment (Zxilly) [reply setup.py:106]: `tests` folder didn't have file `__init__.py`, so it won't be regarded as a module or package.

Comment (fenixpapu) [reply setup.py:106]: I'm newbiew in python.
Thank your answer.

**Comment by hsluoyz** [comment]
@zhangbailong945

**Comment by hsluoyz** [comment]
@zhangbailong945 can you review the PR?

**Pull request merged**